### DISCUSSION TITLE
Impressed

### DISCUSSION THREAD

**Discussion by JLLeitschuh** [ajalt/mordant#147] [General]
I just wanted to come here and say that I was attempting to port a CLI application from python's ritch to java, and I am incredibly impressed that this library even exists and has such a rich (pun partially intended) featureset. Really nice job!

Also, it makes sense that, as you @ajalt are also the author of [clikt](https://github.com/ajalt/clikt) you'd also be the author of this wonderful beauty of a project. Just, overall, nice work! 🎉

**Comment by ajalt** [comment]
Thank you, I appreciate the kind words!

As always, feel free to leave any feedback or feature requests.
